saleor/graphql/product/mutations/product/product_create.py

```diff
@@ -165,24 +165,6 @@ def clean_input(cls, info, instance, data):
         clean_seo_fields(cleaned_input)
         return cleaned_input
 
-    @classmethod
-    def get_instance(cls, info, **data):
```

Comment:
Can you provide more details why we drop this part?

Reply:
Since it is "create" mutation, it doesn't have any `id` to serach for, so it always returns `super().get_instance(info, **data)`, therefore redundant. `ProductUpdate` is the only class, which inherits from `ProductCreate`, but it has own `get_instance` implementation.